docs-generation/scripts/batch_translate_docs.py

```
@@ -139,50 +138,52 @@ def strip_ai_preamble(content: str, lang: str) -> str:
 
     return '\n'.join(lines)
 
+def _extract_table_data_rows(table_text: str) -> list[str]:
+    """Return the data rows of a markdown table (lines starting with '|'),
+    excluding the header row and the |---| separator row."""
+    table_lines = [l for l in table_text.split('\n') if l.strip().startswith('|')]
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)